homework5.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+words = input("Enter some words ")
+
+for latter in words:
+    if latter.isdigit():
+        '''
+        Check later for even or odd
+        '''
+        if int(latter) % 2 == 0:
+            print(f"A difit {latter} is even")
+        elif int(latter) % 2 != 0:
```

Comment:
You can use `else` here since a number can be only even or odd (binary condition). This means that if a number is not ever it is definitely odd